hackage-security/src/Hackage/Security/Trusted/TCB.hs

```diff
@@ -291,8 +301,9 @@ verifyRole' (trusted -> RoleSpec{roleSpecThreshold = KeyThreshold threshold, ..}
       -- was invalid we would already have thrown an error constructing Signed.
       -- (Similarly, if two signatures were made by the same key, the FromJSON
       -- instance for Signatures would have thrown an error.)
-      unless (length (filter isRoleSpecKey sigs) >= fromIntegral threshold) $
-        throwError $ VerificationErrorSignatures targetPath
+      let nSigs = length (filter isRoleSpecKey sigs)
```

Comment:
Why was this `let` binding introduced?  `nSigs` is just used only once later...
Did you men to let-bind `fromIntegral threshold` instead?---this one is used twice.

Reply:
To give it a more descriptive name. I found it helpful when reading the code.